awswrangler/athena/_write_iceberg.py

```diff
@@ -239,6 +239,30 @@ def _validate_args(
         )
 
 
+def _create_partitioned_dataframes(
+    df: pd.DataFrame, partition_cols: list[str] | None, chunk_size: int
```

Comment:
Nit - you can pass default `chunk_size: int=100` here

Reply:
Made the changes